Build failure: vice

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

To reproduce:

1. `nix build 'github:nixos/nixpkgs/nixpkgs-unstable#vice'`

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/296002648

### Relevant log output

```console
error: builder for '/nix/store/63kp24v1j1ijkjmwcj7k17gxfbi16jsv-vice-3.9.drv' failed with exit code 1;
       last 25 log lines:
       > checking for gethostbyname in -lnet... no
       > checking for gethostbyname in -linet... no
       > checking for netinet/tcp.h... yes
       > checking for socket... yes
       > checking for send... yes
       > checking for bind... yes
       > checking for listen... yes
       > checking for gethostbyname... yes
       > checking for connect... yes
       > checking for recv... yes
       > checking for accept... yes
       > checking for htons... yes
       > checking for htonl... yes
       > checking for getdtablesize... yes
       > checking for getrlimit... yes
       > checking For socklen_t... yes
       > checking for in_addr_t... yes
       > checking for Unix domain sockets... yes
       > checking if IPV6 should be enabled... yes
       > checking struct sockaddr::ss_family... yes
       > checking for gethostbyname2... yes
       > checking for FFMPEG support... disabled
       > checking if we are using glibc-2.20 and up... yes
       > checking for pulse/simple.h... no
       > configure: error: pulse audio support missing
```

### Additional context

I guess they added a dependency on pulseaudio for this package, but I'm not sure if anything else is additionally broken

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.14.2, NixOS, 25.05 (Warbler), 25.05.20250417.b024ced`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.1`
 - channels(root): `""`
 - nixpkgs: `/nix/store/isfbldda5j8j6x3nbv1zim0c0dpf90v8-source`

### Notify maintainers


@svanderburg 
---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/vi/vice/package.nix
+++ b/pkgs/by-name/vi/vice/package.nix
@@ -24,6 +24,7 @@
   wrapGAppsHook3,
   xdg-utils,
   libevdev,
+  pulseaudio,
 }:
 
 stdenv.mkDerivation rec {
@@ -60,6 +61,7 @@ stdenv.mkDerivation rec {
     xa
     xdg-utils
     libevdev
+    pulseaudio
   ];
   dontDisableStatic = true;
   configureFlags = [

